library/src/morpheus-master/sass/modules/tool/chat/_chat_variables.scss

```diff
@@ -11,8 +11,8 @@ $footerApp__chat__background: 	 		var(--tool-background-color);
 $footerApp__chat__a__color: 	 		var(--sakai-primary-color-1);
 $footerApp__chat__a--hover__color: 		var(--sakai-primary-color-2);
 
-$footerApp__chat__header__color: 	  	var(--sakai-text-color-1);
-$footerApp__chat__header__background: 	var(--sakai-background-color-1);
+$footerApp__chat__header__color: 	  	var(--sakai-text-color-chat-1);
+$footerApp__chat__header__background: 	var(--sakai-background-color-chat-1);
```

Comment:
I think we could use these variables instead of creating additional ones in the light/dark theme files.
```
$footerApp__chat__a__color: 	 		var(--button-primary-text-color);
$footerApp__chat__a--hover__color: 		var(--button-primary-hover-text-color);

$footerApp__chat__header__color: 	  	var(--button-primary-text-color);
$footerApp__chat__header__background: 	var(--button-primary-hover-text-color);
```

Reply:
typo in my comment, should probably go with 
`$footerApp__chat__header__background: 	var(--button-primary-background);`